core/src/test/scala/integration/kafka/network/DynamicConnectionQuotaTest.scala

```diff
@@ -317,6 +325,12 @@ class DynamicConnectionQuotaTest extends BaseRequestTest {
     }
   }
 
+  private def waitForConnectionCloseToInitState(initialConnectionCount: Int): Unit = {
```

Comment:
How about calling this `waitForConnectionCount` and renaming `initialConnectionCount` to `expectedConnectionCount`? That makes the helper more generic.

Reply:
I'm good. I don't have any preference. Updated. Thanks.